src/backend/distributed/commands/table.c

```diff
@@ -1469,6 +1618,31 @@ PostprocessAlterTableStmt(AlterTableStmt *alterTableStatement)
 			}
 		}
 	}
+
+	/* for the new sequences coming with this ALTER TABLE statement */
+	if (ShouldSyncTableMetadata(relationId))
+	{
+		List *sequenceCommandList = NIL;
+
+		/* if the table is owned by an extension we only propagate pg_dist_* records */
+		bool tableOwnedByExtension = IsTableOwnedByExtension(relationId);
+		if (!tableOwnedByExtension)
```

Comment:
Not sure we should have this check here. Normally we do this to support calling create_distributed_table on a table owned by an extension, which means we should not attempt to create the table on the workers since it would have already come from the extension. If you then attempt to alter the table, I guess we should do it everywhere.